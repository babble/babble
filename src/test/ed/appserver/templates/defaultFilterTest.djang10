{{ unknownVar|default:"should see me" }}
{{ unknownVar|default:a }}
{{ urlParam|urlencode }}
{% filter urlencode %}{{ urlParam }}{% endfilter %}

{{ date|date:"a,A,b,d,D,f,F,g,G,h,H,i,j,l,L,m,M,n,N,O,P,r,s,S,t,w" }}
a   ,A ,b  ,d ,D  ,f   ,F       ,g,G ,h ,H ,i ,j ,l     ,L    ,m ,M  ,n ,N   ,O    ,P        ,r                              ,s ,S ,t ,w

{{ "moo"|upper }}
{{ "MOO"|lower }}

{% if array|length_is:"5" %}should see me{% endif %}

{{ array|length }} 