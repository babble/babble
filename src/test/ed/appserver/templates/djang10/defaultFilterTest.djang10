{% comment %}
    Copyright (C) 2008 10gen Inc.
  
    This program is free software: you can redistribute it and/or  modify
    it under the terms of the GNU Affero General Public License, version 3,
    as published by the Free Software Foundation.
  
    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.
  
    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
{% endcomment %}

{% autoescape off %}

{{ unknownVar|default:"should see me" }}
{{ unknownVar|default:a }}
{{ urlParam|urlencode }}
{% filter urlencode %}{{ urlParam }}{% endfilter %}
{{ urlParam         |             escape }}

{{ date|date:"a,A,b,d,D,f,F,g,G,h,H,i,j,l,L,m,M,n,N,O,P,r,s,S,t,w" }}
a   ,A ,b  ,d ,D  ,f   ,F       ,g,G ,h ,H ,i ,j ,l     ,L    ,m ,M  ,n ,N   ,O    ,P        ,r                              ,s ,S ,t ,w

{{ "moo"|upper }}
{{ "MOO"|lower }}

{% if array|length_is:"5" %}should see me{% endif %}

{{ array|length }} 

{{ "\n\\'\"\t\f"|escapejs }}

{% endautoescape %}
