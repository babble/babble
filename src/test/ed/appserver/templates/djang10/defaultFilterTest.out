should see me
A
%3F%5C/%7E%21.%2C%26%3C%3E
%3F%5C/%7E%21.%2C%26%3C%3E
?\/~!.,&amp;&lt;&gt;

MOO
moo
should see me
5

\n\\\'\"\t\f

1-2-3

37

7

yep
yep

0 bytes
1 KB
1 MB
1 GB

0 votes
1 vote
2 votes
0 classes
1 class
2 classes
0 candies
1 candy
2 candies

A Moo And 1Ba37Ao

<a>w1 w2 ...</a>
<a>w1 w2<b>w3</b></a>
<a>w1 w2<b>w3</b></a>
<a>w1 ...</a>